akka-typed/src/main/scala/akka/typed/internal/SystemMessage.scala

```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed
+package internal
+
+import scala.annotation.tailrec
+
+/**
+ * INTERNAL API
+ *
+ * Helper companion object for [[akka.dispatch.sysmsg.LatestFirstSystemMessageList]] and
+ * [[akka.dispatch.sysmsg.EarliestFirstSystemMessageList]]
+ */
+private[typed] object SystemMessageList {
+  final val LNil: LatestFirstSystemMessageList = new LatestFirstSystemMessageList(null)
```

Comment:
yeah, I got it, just the coment above referring to akka.dispatch.sysmsg confused me